ui-v2/tests/acceptance/components/intention-filter.feature

```diff
@@ -0,0 +1,55 @@
+@setupApplicationTest
+Feature: components / intention filter: Intention Filter
+  In order to find the intention I'm looking for easier
+  As a user
+  I should be able to filter by 'policy' (allow/deny) and freetext search tokens by source and destination
+  Scenario: Filtering [Model]
+    Given 1 datacenter model with the value "dc-1"
+    And 2 [Model] models
+    When I visit the [Page] page for yaml
+    ---
+      dc: dc-1
+    ---
+    Then the url should be [Url]
+
+    Then I see 2 [Model] models
+    And I see allIsSelected on the filter
+
+    When I click allow on the filter
+    Then I see allowIsSelected on the filter
+    And I see 1 [Model] model
+    And I see 1 [Model] model with the action "allow"
+
+    When I click deny on the filter
+    Then I see denyIsSelected on the filter
+    And I see 1 [Model] model
+    And I see 1 [Model] model with the action "deny"
+
+    When I click all on the filter
+    Then I see 2 [Model] models
+    Then I see allIsSelected on the filter
+    Then I fill in with yaml
```

Comment:
Not sure I understand this comment. Basically it clicks the [All] filter button, makes sure both intentions are visible, then makes sure the [All] button is in the selected state, then fills 'alarm' into the search box, then checks there is only 1 intention visible and that there is 1 intention with the SourceName as 'alarm'. Shout me later and we can chat.

Reply:
> then fills 'alarm' into the search box

That was the part I was asking about, because "Then I fill in with yaml" doesn't explicitly say you are filling in the search box.

Reply:
Ah gotcha:

https://github.com/hashicorp/consul/blob/83caa6a2966bc38fdd8f2c78e331977360db3ae1/ui-v2/tests/acceptance/components/intention-filter.feature#L31-L34

It takes a yaml (or json) blob which just gets looped through and passed to `fillIn`

so `s` is the `name` attribute of the search `<input>`, I think `fillIn` can take other things as well though.